frontend/src/api/prometheus/serving.ts

```diff
@@ -116,6 +116,43 @@ export const useModelServingMetrics = (
     PROMETHEUS_BIAS_PATH,
   );
 
+  const modelRequestCount = useQueryRangeResourceData(
+    performanceMetricsAreaAvailable && type === PerformanceMetricType.MODEL,
+    (queries as { [key in ModelMetricType]: string })[ModelMetricType.REQUEST_COUNT],
+    end,
+    timeframe,
+    defaultResponsePredicate,
+    namespace,
+  );
+
+  const modelAverageResponseTime =
+    useQueryRangeResourceData<PrometheusQueryRangeResponseDataResult>(
+      performanceMetricsAreaAvailable && type === PerformanceMetricType.MODEL,
+      (queries as { [key in ModelMetricType]: string })[ModelMetricType.AVG_RESPONSE_TIME],
+      end,
+      timeframe,
+      prometheusQueryRangeResponsePredicate,
+      namespace,
+    );
+
+  const modelCPUUtilization = useQueryRangeResourceData(
+    performanceMetricsAreaAvailable && type === PerformanceMetricType.MODEL,
+    (queries as { [key in ModelMetricType]: string })[ModelMetricType.CPU_UTILIZATION],
+    end,
+    timeframe,
+    defaultResponsePredicate,
+    namespace,
+  );
+
+  const modelMemoryUtilization = useQueryRangeResourceData(
+    performanceMetricsAreaAvailable && type === PerformanceMetricType.MODEL,
+    (queries as { [key in ModelMetricType]: string })[ModelMetricType.MEMORY_UTILIZATION],
+    end,
+    timeframe,
+    defaultResponsePredicate,
+    namespace,
+  );
```

Comment:
We won't actually need these kinds of queries - the kserve ones are going to be dynamically pulled from a configmap on the cluster, as such there's going to be a wrapper component that handles fetching the data for each chart.